README.md

```diff
@@ -0,0 +1,54 @@
+# Inbox - The simplest Aptos backend
+
+Inbox is a simple backend server that automatically creates a REST API for your
+contract using your events.
+
+## Dependencies
+
+The only dependency you need is Docker.
+
+## Setting up
+
+### Step 0: Cloning
+
+First, clone this project: `git clone https://github.com/CRBl69/inbox`.
+
+Then, clone git submodules: `git submodule update --init --recursive`.
```

Comment:
Change to new repo link? or whatever it needs to be now that it's in `econia-labs`